src/frontend/apps/drive/src/features/explorer/components/tree/ExplorerTreeItem.tsx

```diff
@@ -80,3 +64,33 @@ export const ExplorerTreeItem = ({ ...props }: ExplorerTreeItemProps) => {
     </>
   );
 };
+
+export const ExplorerTreeItemIcon = ({
```

Comment:
Apart from the folder icon, other icons are also used, for example in the breadcrumb. Can the component be renamed and used everywhere?